app/screens/TalkDetailsScreen/AssistantsList.tsx

```diff
@@ -1,11 +1,14 @@
 import React from "react"
-import { ImageStyle, TextStyle, View, ViewStyle } from "react-native"
+import { Dimensions, ImageStyle, TextStyle, View, ViewStyle } from "react-native"
 import { AutoImage, Text } from "../../components"
 import { translate } from "../../i18n"
 import { colors, spacing } from "../../theme"
 import { Speaker } from "../../services/api/webflow-api.types"
 import { SocialButtons } from "../../components/SocialButton"
 
+const ASSISTANT_GAP = spacing.medium
+const ASSISTANT_WIDTH = (Dimensions.get("window").width - ASSISTANT_GAP * 3) / 2
```

Comment:
This doesn't account for the padding of the container. Based on the calculation, there is no gap as 3 * 16 is 48. This is the padding of the container making the gap 0. Any reason for us not to use % for width?

Reply:
great point, not sure why i did this in the first place 🤔 updated in next push